examples/load-reporting-service/server/lrs_server.go

```diff
@@ -0,0 +1,161 @@
+package server
+
+import (
+	"context"
+	"github.com/golang/protobuf/ptypes/duration"
+	"google.golang.org/grpc"
+	"log"
+	"sync/atomic"
+
+	core "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
+	gcpLoadStats "github.com/envoyproxy/go-control-plane/envoy/service/load_stats/v2"
+)
+
+type NodeMetadata struct {
+	stream stream
+	node   *core.Node
+}
+
+type Server interface {
+	gcpLoadStats.LoadReportingServiceServer
+	SendResponse(cluster string, upstreamCluster []string, frequency int64)
+}
+
+// Callbacks is a collection of callbacks inserted into the server operation.
+// The callbacks are invoked synchronously.
+type Callbacks interface {
```

Comment:
i don't think 'Callbacks' adds much to the example and I would consider removing them.